src/styles/homestyle/style.css

```diff
@@ -0,0 +1,375 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Oxygen:wght@300;400;700&display=swap');
+* {box-sizing: border-box;}
+
+body { 
+  margin: 0;
+ font-family: 'Oxygen', sans-serif;
+  overflow-x: hidden;
+}
+/*header*/
+.header {
+  overflow: hidden;
+  background-color: #fff;
+  padding: 20px 10px;
+  text-decoration: none;
+  padding: 20;
+    position: fixed;
+    width: 100%;
+    left: 0;
+    right: 50;
+    -webkit-box-shadow: 0 7px 13px 0 rgba(0, 0, 0, 0.1);
+    box-shadow: 0 7px 13px 0 rgba(0, 0, 0, 0.1);
+    top: 0;
+    z-index: 999;
+    padding: 15px 0;
+    
+    
+}
+
+.header a {
+  float: left;
+  text-align: center;
+  padding: 8px 15px;
+  text-decoration: none !important;
+  line-height: 25px;
+  border-radius: 4px;
+  font-size: 18px;
+  font-weight: 500;
+  color: #283a5e;
+}
+
+.header a.logo {
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.header a:hover {
+  background-color: #01358D;
+  color: #fff;
+}
+
+.header a.active {
+  background-color: dodgerblue;
+  color: white;
+}
+
+.header-right {
+  float: right;
+}
+@media screen and (max-width: 500px) {
+  .header a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
+  
+  .header-right {
+    float: none;
+  }
+}
+/* The hero image header banner*/
+.hero-image {
+  background-color:#01358D;
+  height: 600px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+.header-text-box {
+  margin-top:150px !important;
+}
+.header-text-box p{
+  color:#fff;
+  font-size: 25px;
+}
+.header-text-box h1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size:78px;
+  line-height:78px;
+  font-weight: 900;
+  color:#fff;
+  margin-top:150px;
+}
+.hero-text {
+  text-align: left;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+}
+.hero-text h1 {
+ font-family: 'Oxygen', sans-serif;
+  font-size: 75px !important;
+  line-height: 54px !important;
+  font-weight: 700;
+}
+.hero-text p {
+font-size:25px;
+font-family: 'DM Sans', sans-serif;
+}
+.btnbanner{
+ 
+  border-radius: 4px;
+ 
+  background-color: transparent;
+  font-weight: 400;
+  height: 60px !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)